docs/_source/_common/next_steps.html

```diff
@@ -0,0 +1,10 @@
+<section id="Next-steps">
+    <h2>Next steps<a class="headerlink" href="#Next-steps" title="Permalink to this headline">#</a></h2>
+    <p>⚙️ <a class="reference internal" href="#setup-argilla">The section</a> about setting up Argilla and opening or
+        downloading the notebook.</p>
+    <p>📚 <a class="reference external" href="https://docs.argilla.io">Argilla documentation</a> for more guides and
+        tutorials.</p>
+    <p>🙋‍♀️ Join the Argilla community! A good place to start is the <a class="reference external"
```

Comment:
I would add the join Slack community paragraph from this tutorial: https://docs.argilla.io/en/latest/tutorials/notebooks/labelling-textclassification-sentence-transformers-semantic.html#Next-steps, before the Argilla documentation paragraph.

Let's discuss briefly about the first paragraph about setup